docs(flutter): Add  `Rewrite Frames` documentation (#10470)
getsentry/sentry-docs: docs/platforms/flutter/configuration/rewriteframes.mdx

---
title: Rewrite Frames
sidebar_order: 70

> description: "Learn more about rewriting frames of the stack trace."

Reviewer: ```suggestion
description: "Learn how and why to rewrite stack trace frames in Sentry's flutter SDK."
```

---

> There are instances where you might want to rewrite frames of a stack trace before sending it to Sentry. For example, when you are publishing your Flutter application on the web and want to set a `url_prefix` because it is not deployed at the root of your URL. In this case, the absolute path of your stack frames also needs to include the same prefix so that the source maps can be found for deobfuscation. To set the `url_prefix`, you can use the [Sentry Dart Plugin](https://github.com/getsentry/sentry-dart-plugin). Below is a sample of how to update the stack frame's absolute path to include the prefix using the <PlatformLink to="/configuration/options/#before-send"><PlatformIdentifier name="before-send" /></PlatformLink> hook.

Reviewer: ```suggestion
There are times when you may want to rewrite stack trace frames before sending them to Sentry. For example, if you're publishing your Flutter application on the web and want to set a `url_prefix` because it's not deployed at the root of your URL. In this case, the absolute path of your stack frames also needs to include the same prefix so that the source maps can be found for deobfuscation. 

To set the `url_prefix`, use the [Sentry Dart Plugin](https://github.com/getsentry/sentry-dart-plugin). Below is an example of how to update the stack frame's absolute path to include the prefix using the <PlatformLink to="/configuration/options/#before-send"><PlatformIdentifier name="before-send" /></PlatformLink> hook.
```
